.githooks/pre-commit

```diff
@@ -1,8 +1,39 @@
 #!/bin/bash
 cd "$(git rev-parse --show-toplevel)"
+# Pylint Path is Micky's local path, feel free to change in PR 
+PYLINT = "c:\users\micky\appdata\local\programs\python\python310\lib\site-packages" 
```

Comment:
I think you have to point to site-packages\pylint or whatever the directory is called in your instance